pl_bolts/models/diffeq/neuralode.py

```diff
@@ -0,0 +1,115 @@
+from argparse import ArgumentParser
+
+import pytorch_lightning as pl
+import torch
+import torch.nn as nn
+from torch import Tensor
+
+from torchdyn.models import NeuralDE, DataControl
+
```

Comment:
We need to import optional packages as the following to avoid import errors when optional packages are not installed. I think this should solve the import errors in the doctest.
```suggestion

from pl_bolts.utils import _TORCHDYN_AVAILABLE
from pl_bolts.utils.warnings import warn_missing_pkg

if _TORCHDYN_AVAILABLE:
    from torchdyn.models import NeuralDE, DataControl
else:  # pragma: no cover
    warn_missing_pkg("torchdyn")
```

In addition to this suggestion, could you also define `_TORCHDYN_AVAILABLE` in `pl_bolts/utils/__init__.py`.